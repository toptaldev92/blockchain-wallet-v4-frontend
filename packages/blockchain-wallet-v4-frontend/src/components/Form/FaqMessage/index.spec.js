import { shallow } from 'enzyme'
import FaqMessage from './index.js'
import React from 'react'
import toJson from 'enzyme-to-json'

describe('FooterShadowWrapper', () => {
  it('renders correctly', () => {
    const props = { title: <div />, text: <div />, className: 'cn' }
    const component = shallow(<FaqMessage {...props} />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
