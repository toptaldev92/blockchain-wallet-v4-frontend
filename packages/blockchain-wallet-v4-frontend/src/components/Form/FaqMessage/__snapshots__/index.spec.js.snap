// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterShadowWrapper renders correctly 1`] = `
<styled.div
  className="cn"
>
  <Styled(Icon) />
  <Styled(Text)>
    <div />
  </Styled(Text)>
  <Styled(Text)>
    <div />
  </Styled(Text)>
</styled.div>
`;
